packages/rOpenScPCA/R/calculate-clusters.R

```diff
@@ -0,0 +1,115 @@
+#' Calculate graph-based clusters from a provided matrix
+#'
+#' @param mat Matrix, usually of PCs, where each row is a cell. Matrix must have rownames of cell ids (e.g., barcodes)
+#' @param algorithm Clustering algorithm to use. Must be one of "louvain" (default), "walktrap", or "leiden".
+#'  Be aware that the default of "louvain" is different from the bluster package default of "walktrap". This difference is
+#'  because louvain clustering is more commonly-used in scRNA-seq analysis.
```

Comment:
I would actually leave off the explanation here. Instead, I was suggesting a paragraph of explanation before the `@params`, which will end up in the function description.